desc/magnetic_fields.py

```diff
@@ -1803,6 +1810,299 @@ def from_surface(
         )
 
 
+class OmnigenousField(Optimizable, IOAble):
+    """Model field with perfect omnigenity.
+
+    Notes
+    -----
+    Doesn't conform to MagneticField API, as it only knows about :math:`|B|` in flux
+    coordinates, not vector B in lab coordinates.
+
+    """
+
+    _io_attrs_ = [
+        "_L_well",
+        "_M_well",
+        "_L_shift",
+        "_M_shift",
+        "_N_shift",
+        "_NFP",
+        "_helicity",
+        "_well_basis",
+        "_shift_basis",
+        "_B_lm",
+        "_x_lmn",
+    ]
+
+    def __init__(
+        self,
+        L_well=0,
+        M_well=2,
+        L_shift=0,
+        M_shift=0,
+        N_shift=0,
+        NFP=1,
+        helicity=(1, 0),
+        B_lm=None,
```

Comment:
L_well and M_well correspond to B_lm right? why not make this `well_lm` instead of `B_lm`? would at least make the resolutions clearer as to what they correspond to